### PR TITLE
release version 2.1.0

### PR DESCRIPTION
**Pull request by jowong4**
Making a new release for RNA and multi hash per seed support.

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+2020-01-21  Johnathan Wong <jowong@bcgsc.ca>
+
+	* Release version 2.1.0
+	* Added support for RNA sequences.
```
Comment (lcoombe) [inline ChangeLog]:
```suggestion
	* Added support for RNA sequences
```

File: ChangeLog [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+	* Updated stHashIterator to produce multiple hashes for each seeds.
```
Comment (lcoombe) [inline ChangeLog]:
```suggestion
	* Updated stHashIterator to produce multiple hashes for each seed
```
Is this the correct meaning? Or `for all seeds`?

Comment (jowong4) [reply ChangeLog]: `Updated stHashIterator with the ability to produce multiple hashes for each seed` Is this better?

Comment (jowong4) [reply ChangeLog]: Also with regards to the period, I see Ben using them in his release notes

Comment (lcoombe) [reply ChangeLog]: I don't normally put periods with bullet points, but I don't care too much either way - let's just be consistent (ie. first bullet point didn't have a period)

Comment (lcoombe) [reply ChangeLog]: Hmm..
```
stHashIterator is now able to produce multiple hashes for each seed
```
?

**Pull request merged**